src/Microsoft.Health.Fhir.Liquid.Converter.Tool.UnitTests/ConverterLogicHandlerTest.cs

```diff
@@ -10,92 +11,108 @@ public class ConverterLogicHandlerTest
     [Fact]
     public void ConvertWithoutSaving_ShouldThrowDirectoryNotFoundException_WhenTemplateDirectoryDoesNotExist()
     {
-        var templateDirectory = "nonexistent_directory";
-        var rootTemplate = "rootTemplate";
-        var inputDataContent = "inputDataContent";
-        var isVerboseEnabled = false;
-        var isTraceInfo = false;
+        var options = new ConverterOptions()
+        {
+            TemplateDirectory = "nonexistent_directory",
+            RootTemplate = "rootTemplate",
+            InputDataContent = "inputDataContent",
+            IsVerboseEnabled = false,
+            IsTraceInfo = false,
```

Comment:
My read is these two default false - are they here to make it easier to toggle? Or is my read wrong?